app/carbonapi/http_handlers.go

```diff
@@ -127,8 +127,9 @@ func initHandlers(app *App) http.Handler {
 	return r
 }
 
-func writeResponse(w http.ResponseWriter, b []byte, format string, jsonp string) {
+func writeResponse(w http.ResponseWriter, b []byte, format string, jsonp string, ctx context.Context) {
 
+	w.Header().Set("X-carbonapi-uuid", util.GetUUID(ctx))
```

Comment:
I think, "X-Carbonapi-UUID" would be a more canonical name.

Reply:
I agree